awx/ui_next/src/routeConfig.js

```diff
@@ -0,0 +1,181 @@
+import { t } from '@lingui/macro';
+
+import Applications from './screens/Application';
+import Credentials from './screens/Credential';
+import CredentialTypes from './screens/CredentialType';
+import Dashboard from './screens/Dashboard';
+import Hosts from './screens/Host';
+import InstanceGroups from './screens/InstanceGroup';
+import Inventory from './screens/Inventory';
+import InventoryScripts from './screens/InventoryScript';
+import { Jobs } from './screens/Job';
+import ManagementJobs from './screens/ManagementJob';
+import NotificationTemplates from './screens/NotificationTemplate';
+import Organizations from './screens/Organization';
+import Portal from './screens/Portal';
+import Projects from './screens/Project';
+import Schedules from './screens/Schedule';
+import AuthSettings from './screens/AuthSetting';
+import JobsSettings from './screens/JobsSetting';
+import SystemSettings from './screens/SystemSetting';
+import UISettings from './screens/UISetting';
+import License from './screens/License';
+import Teams from './screens/Team';
+import Templates from './screens/Template';
+import Users from './screens/User';
+
+// Ideally, this should just be a regular object that we export, but we
```

Comment:
I think we need to start tracking these comments in the code with an issue.  I realize that when we adopt lingui3 this will be more of an enhancement, but it would be good to track.

Reply:
@AlexSCorey ok. done. https://github.com/ansible/awx/issues/7215